modules/apps/portal-security/portal-security-ldap-impl/src/main/java/com/liferay/portal/security/ldap/internal/configuration/persistence/listener/LDAPImportConfigurationModelListener.java

```diff
@@ -40,6 +40,7 @@
  * @author Mariano Álvaro Sáiz
  */
 @Component(
+	enabled = true,
```

Comment:
First, "enable=true" is the default value, no need to set.
Second, please do not have a separate commit to modify a class you just added, please modify it directly in the commit you added.
Third, a component with "enable=true" does not mean it will be activated immediately, you should use immediate=true, I do not understand why it was disabled and then you enable it here.